pkg/rsyslog/build.yml

```diff
@@ -0,0 +1,13 @@
+image: eve-rsyslog
+org: lfedge
+network: yes
+config:
+  binds:
+    - /dev:/dev
```

Comment:
We should restrict the list of static binds to only what is currently needed. I suspect we don't need any of this for now.

Reply:
I'll remove these.